Backend/dev/migrations/dynamic-offer-driver-app/0155-add-rating-categories.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE atlas_driver_offer_bpp.rating_category (
+  id character varying(255) NOT NULL,
+  category character varying(255) NOT NULL,
+  CONSTRAINT rating_category_pkey PRIMARY KEY (id)
+);
+
+INSERT INTO atlas_driver_offer_bpp.rating_category (id, category) VALUES (atlas_driver_offer_bpp.uuid_generate_v4(), 'RIDE');
+
+CREATE TABLE atlas_driver_offer_bpp.feedback_form (
+  id character varying(255) NOT NULL,
```

Comment:
it should be` character (36)` I guess

Reply:
I meant `character (36)`, not `character varying (36)` because `uuid_generate_v4` generates exactly 36 characters